packages/react-frontend/src/apis.js

```diff
@@ -33,4 +33,19 @@ function postQuestion(question) {
   return promise;
 }
 
-export { fetchComments, fetchQuestions, getQuestion, postQuestion };
+function postComment(comment, questionID) {
+  const promise = fetch(`${url}/questions/${questionID}/comments`, {
```

Comment:
Not for this CR, but after enabling frontend proxy for the backend, we should probably put a `/api/` in the backend endpoints.

Reply:
Sounds good